external/storm-cassandra/src/main/java/org/apache/storm/cassandra/client/CassandraConf.java

```diff
@@ -0,0 +1,146 @@
+    public static final String CASSANDRA_KEYSPACE           = "cassandra.keyspace";
+    public static final String CASSANDRA_CONSISTENCY_LEVEL  = "cassandra.output.consistencyLevel";
+    public static final String CASSANDRA_NODES              = "cassandra.nodes";
+    public static final String CASSANDRA_PORT               = "cassandra.port";
+    public static final String CASSANDRA_BATCH_SIZE_ROWS    = "cassandra.batch.size.rows";
+
+    /**
+     * The authorized cassandra username.
+     */
+    private String username;
+    /**
+     * The authorized cassandra password
+     */
+    private String password;
+    /**
+     * The cassandra keyspace.
+     */
+    private String keyspace;
+    /**
+     * List of contacts nodes.
+     */
+    private String[] nodes = {"localhost"};
+
+    /**
+     * The port used to connect to nodes.
+     */
+    private int port = 9092;
+
+    /**
+     * Consistency level used to write statements.
+     */
+    private ConsistencyLevel consistencyLevel = ConsistencyLevel.ONE;
+    /**
+     * The maximal numbers of rows per batch.
+     */
+    private int batchSizeRows       = 100;
+    
+    /**
+     * Creates a new {@link CassandraConf} instance.
+     */
+    public CassandraConf() {
+        super();
+    }
+
+    /**
+     * Creates a new {@link CassandraConf} instance.
+     *
+     * @param conf The storm configuration.
+     */
+    public CassandraConf(Map<String, Object> conf) {
+        this.username = getOrElse(conf, CASSANDRA_USERNAME, null);
+        this.password = getOrElse(conf, CASSANDRA_PASSWORD, null);
+        this.keyspace = get(conf, CASSANDRA_KEYSPACE);
+        this.consistencyLevel = ConsistencyLevel.valueOf(getOrElse(conf, CASSANDRA_CONSISTENCY_LEVEL, ConsistencyLevel.ONE.name()));
+        this.nodes    = getOrElse(conf, CASSANDRA_NODES, "localhost").split(",");
+        this.batchSizeRows = getOrElse(conf, CASSANDRA_BATCH_SIZE_ROWS, 100);
+        this.port = conf.get(CASSANDRA_PORT) != null ? Integer.valueOf((String)conf.get(CASSANDRA_PORT)) : 9042;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public String getKeyspace() {
+        return keyspace;
+    }
+
+    public String[] getNodes() {
+        return nodes;
+    }
+
+    public ConsistencyLevel getConsistencyLevel() {
+        return consistencyLevel;
+    }
+
+    public int getBatchSizeRows() {
+        return batchSizeRows;
+    }
+
+    public int getPort() {
+        return this.port;
+    }
+
+    private <T> T get(Map<String, Object> conf, String key) {
+        Object o = conf.get(key);
+        if(o == null) {
+            throw new IllegalArgumentException("No '" + key + "' value found in configuration!");
+        }
+        return (T)o;
+    }
+
+    private <T> T getOrElse(Map<String, Object> conf, String key, T def) {
```

Comment:
That does pretty much the same as [https://github.com/apache/storm/blob/768a85926373355c15cc139fd86268916abc6850/storm-core/src/jvm/backtype/storm/utils/Utils.java#L344], doesn't it?